src/client/scripts/game/chess/gamefile.js

```diff
@@ -172,6 +172,32 @@ function gamefile(metadata, { moves = [], variantOptions, gameConclusion } = {})
 	/** Contains the methods for undo'ing special moves for this game. */
 	this.specialUndos = undefined;
 
+	/** The time each player has remaining, in milliseconds. */
+	this.currentTime = {
+		white: undefined,
+		black: undefined,
```

Comment:
All of these clock-related variables can you place together inside an object say, "clock"? Similar to how mesh data is all together.

Reply:
https://github.com/Infinite-Chess/infinitechess.org/pull/282/commits/8aa01c2c05a1b8af2597241946e6cbf1aaa3159
Clock data is now contained under `gamefile.clocks`